uwblueprint/social-diversity-for-children
# feat(seeding): add sample seeding for prod and dev

### Notion ticket link



[Seeding Database](https://www.notion.so/uwblueprintexecs/Seeding-Database-32402df8657d407285565f2976f4c009)



### Implementation description

- Add sample seeding for classes, programs, and program translations which refects the records in staging.



### Steps to test

1. Create a new DB URL
2. run `yarn`
2. run the sql script against the DB (using the heroku client)
3. do prisma introspect and generate
4. run `npx ts-node -O {\"module\":\"CommonJS\"} prisma/seed.ts` 
5. run `yarn dev` and ensure pages with program are populated



### What should reviewers focus on?

- content of sample data
- how seeding is done
- style

### Checklist
